### PR TITLE
basic frontend

### PR DESCRIPTION
**Pull request by jspaaks**
~~(Can be marked as ready for review once #454 is merged)~~

This PR adds a very simple frontend page for the projects.

Check for example http://localhost/projects/1403

Refs #461

### CODE REVIEW

**Comment by jspaaks** [comment]
The live tests fail as expected because `https://www.research-software.nl/api` serves data of a different version than what this PR expects (created #471).

File: frontend/templates/software/project_template.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {{super()}}
+    <title>{{template_data.title}} | Research Software Directory</title>
+    <meta property="description" content="{{template_data.description}}" />
+    <meta property="og:title" content="{{template_data.title}}" />
+    <meta property="og:description" content="{{template_data.description}}" />
+    <meta property="og:url" content="https://www.research-software.nl/projects/{{template_data.id}}" />
+    <meta property="og:site_name" content="Research Software Directory" />
+    <script type="text/javascript" src="{{url_for('static', filename='scripts/polyfills.js')}}"></script>
+    <script type="text/javascript" src="{{url_for('static', filename='scripts/software_scripts.js')}}"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.9/vue.min.js" crossorigin="anonymous"></script>
```
Comment (fdiblen) [inline frontend/templates/software/project_template.html:12]:
```suggestion

```

**Review by fdiblen — APPROVED** [review]
:+1:

**Comment by jspaaks** [comment]
Thanks Faruk!

**Pull request merged**